### PR TITLE
Add Azure Resources Integration to CLI

### PR DESCRIPTION
**Pull request by taimoorgit**
## Issues

https://github.com/OpsLevel/team-platform/issues/414

https://github.com/OpsLevel/opslevel-go/pull/430

## Changelog

- [x] Create function
- [x] Update function
- [x] Make a `changie` entry

## Tophatting

YAML used:

```
version: 1
kind: aws
spec:
  name: "Prod"
  iamRole: "arn:aws:iam::XXXXX:role/opslevel-integration"
  externalId: "XXXXXX"
  ownershipTagOverride: true
  ownershipTagKeys: ["owner","service","app"]

version: 1
kind: aws
spec:
  name: "dev"
  ownershipTagKeys: ["owner","service","application"]

version: 1
kind: azure
spec:
  name: "Azure New"
  tenantId: "12345678-1234-1234-1234-123456789abc"
  subscriptionId: "12345678-1234-1234-1234-123456789def"
  clientId: "XXX_CLIENT_ID_XXX"
  clientSecret: "XXX_CLIENT_SECRET_XXX"

version: 1
kind: azure
spec:
  name: "Azure Updated"
```

CLI output:

```
$ go run main.go create integration -f aws_create.yaml
Created aws integration 'Prod' with id 'Z2lkOi8vb3BzbGV2ZWwvSW50ZWdyYXRpb25zOjpBd3NJbnRlZ3JhdGlvbi8zNQ'
$ go run main.go update integration Z2lkOi8vb3BzbGV2ZWwvSW50ZWdyYXRpb25zOjpBd3NJbnRlZ3JhdGlvbi8zNQ -f aws_update.yaml
Updated aws integration 'dev' with id 'Z2lkOi8vb3BzbGV2ZWwvSW50ZWdyYXRpb25zOjpBd3NJbnRlZ3JhdGlvbi8zNQ'

$ go run main.go create integration -f azure_create.yaml
Created azure integration 'Azure New' with id 'Z2lkOi8vb3BzbGV2ZWwvSW50ZWdyYXRpb25zOjpBenVyZVJlc291cmNlc0ludGVncmF0aW9uLzM2'
$ go run main.go update integration Z2lkOi8vb3BzbGV2ZWwvSW50ZWdyYXRpb25zOjpBenVyZVJlc291cmNlc0ludGVncmF0aW9uLzM2 -f azure_update.yaml
Updated azure integration 'Azure Updated' with id 'Z2lkOi8vb3BzbGV2ZWwvSW50ZWdyYXRpb25zOjpBenVyZVJlc291cmNlc0ludGVncmF0aW9uLzM2'
```

### CODE REVIEW

**Review by davidbloss — APPROVED** [review]
LGTM 🎉

File: src/cmd/integration.go [hunk 1]
```diff
@@ -57,16 +68,42 @@ spec:
   name: "Prod"
   iamRole: "arn:aws:iam::XXXXX:role/opslevel-integration"
   externalId: "XXXXXX"
-  ownershipTagOverrides: true
+  ownershipTagOverride: true
```
Comment (taimoorgit) [inline src/cmd/integration.go:71]:
```suggestion
  awsTagsOverrideOwnership: true
```

Needs to match the tag in opslevel-go, see here https://github.com/OpsLevel/opslevel-go/blob/408867194d2bb24daf1627d8cb759fdfc24f9089/integration.go#L45

File: src/cmd/integration.go [hunk 2]
```diff
@@ -111,21 +148,46 @@ var updateIntegrationCmd = &cobra.Command{
 	Aliases: []string{"int"},
 	Short:   "Update an integration",
 	Long:    `Update an integration`,
-	Example: `cat << EOF | opslevel update integration XXXXXXXX -f -
+	Example: `cat << EOF | opslevel update integration Z2lkOi8vb123456789 -f -
 version: 1
 kind: aws
 spec:
-  ownershipTagOverrides: true
+  ownershipTagOverride: true
```
Comment (taimoorgit) [inline src/cmd/integration.go:155]:
```suggestion
  awsTagsOverrideOwnership: true
```

**Review by taimoorgit — COMMENTED** [review]
Important changes to example that I noticed was incorrect

**Pull request merged**